.env.example

```diff
@@ -0,0 +1,78 @@
+# ==============================================================================
+# 0. METABASE DEVELOPMENT ENVIRONMENT
+# ==============================================================================
+#
+# Central configuration for local Metabase development.
+# Copy to .env and uncomment/modify values as needed.
+#
+# Usage:
+#   cp .env.example .env                                (populate manually)
+#   op inject --in-file .env.example --out-file .env    (populate using CLI)
+#
+# Hint: Auto-load environment variables with direnv or mise:
+#   direnv  - Create .envrc with: dotenv .env       (https://direnv.net)
+#   mise    - mise.toml: [env] _.file = ".env"      (https://mise.jdx.dev)
+#
+# Full Metabase environment configuration reference:
+# https://www.metabase.com/docs/latest/configuring-metabase/environment-variables
+
+
+# ==============================================================================
+# 1. E2E / CYPRESS TESTING
+# ==============================================================================
+#
+# Configuration for running Cypress end-to-end tests locally.
+#
+# The whole environment is already orchestrated via deps.edn using :e2e alias.
+# Treat it as a source of truth for JVM flags and ENVs whether you decide to run
+# backend on your own, or using `yarn test-cypress` (local test runner).
+#
+# You don't _have to_ export any of these vars (except for the tokens below).
+# Things will still work.
+
+# 1.1. E2E Application Database (H2 required)
+# ------------------------------------------------------------------------------
+
+# Whether to run the enterprise or the open-source Metabase version
+MB_EDITION="ee"
+
+# Which file to use for the ephemeral testing application database?
+MB_DB_FILE="./e2e/tmp/cypress-test.db"
+
+# Where does Cypress go to look for the application?
+MB_JETTY_PORT=4000
+
+# 1.2 E2E Local Runner Setup
+# ------------------------------------------------------------------------------
+
+# Whether to run true e2e test suite, or SDK component tests (e2e | component)
+CYPRESS_TESTING_TYPE="e2e"
+
+# Whether to start Cypress in an interactive (open) mode with full GUI
+CYPRESS_GUI="true"
+
+# Whether to generate application database dumps (colloquially "snapshots")
+# Unless you're experimenting, you generally want to leave this on.
+GENERATE_SNAPSHOTS="true"
```

Comment:
These shouldn't really be environment variable "knobs". But making even more radical changes in a single PR would be too much. I had to draw the line somewhere, and this seemed like an acceptable compromise.

In some near future I hope to have a proper `e2e-setup` automation (most likely via `mage`) where one could control different aspects of the said environment via cli `--flags`.

Once the environment is up, one simply needs to run or open Cypress. Both the "testing type" and the "gui" exist as the Cypress-native CLI arguments.

```
npx cypress open --e2e --config path/to/config
```

or
```
npx cypress run --component --config path/to/config
```